debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerConnectorConfig.java

```diff
@@ -290,10 +299,18 @@ public static SnapshotIsolationMode parse(String value, String defaultValue) {
                     + "In '" + SnapshotIsolationMode.READ_UNCOMMITTED.getValue()
                     + "' mode neither table nor row-level locks are acquired, but connector does not guarantee snapshot consistency.");
 
+    public static final Field TASK_DATABASE_NAMES = Field.create("task.database.dbnames")
```

Comment:
Is this a "surrogate" field only used for connector/task communication? I don't think the user would manually have to assign DBs to tasks?

Reply:
> Is this a "surrogate" field only used for connector/task communication?

Yes.

> I don't think the user would manually have to assign DBs to tasks?

Correct.

What would be the proper way to represent such a field in the Debezium configuration? Maybe move under `internal.`?

We've just attempted to extract this parameter into a separate configuration class (https://github.com/sugarcrm/debezium/pull/43) but it's not currently clear how to properly wire it up with the Connect-level config validation.